docs/source/remote-data-services.rst

```diff
@@ -187,7 +187,13 @@ The following parameters may be passed to s3fs using ``storage_options``:
     - use_ssl: Whether connections are encrypted and secure (default True)
 
     - client_kwargs: Dict passed to the `boto3 client`_, with keys such
-      as `region_name` or `endpoint_url`
+      as `region_name` or `endpoint_url`. Notice: do not pass the `config`
+      option here, which will cause an ``TypeError: client() got multiple
```

Comment:
Do not need to say what the error would be if the user chooses to ignore this message. It may be reasonable, instead, to test in the code for overloaded keywords.

Reply:
ok. The reason I added the error messages here is because I want people who search the google will goes to here. But a clear notice withoug error is enough, too